packages/react-ai/src/components/AIConversation/context/ResponseComponentsContext.tsx

```diff
@@ -4,9 +4,22 @@ import { ToolConfiguration } from '../../../types';
 
 type ResponseComponentsContextProps = ResponseComponents | undefined;
 
+export const RESPONSE_COMPONENT_PREFIX = 'UI_';
```

Comment:
Probably overkill and something i usually nitpick, but should we make this more specific by prefixing `AMPLIFY_` as well?

Reply:
Sure, happy to add